ThunderCats/UIImageExtensionHelper.m

```diff
@@ -0,0 +1,40 @@
+//
+//  UIImageExtensionHelper.m
+//  ThunderCats
+//
+//  Created by William Grand on 5/26/16.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import "UIImageExtensionHelper.h"
+
+@implementation UIImageExtensionHelper
+
++ (UIColor *) createTintEffectColor:(UIColor *)color {
+    const CGFloat EffectColorAlpha = 0.6;
```

Comment:
I know this is from the old version of this code, but I think we should start this variable name with a lower case k as per common with Objective-C/C constants: `kEffectColorAlpha`.